Add JSON Forms editor text (#272)
eclipsesource/jsonforms2-website: content/community/community.mdx

In case you have a question or want to share an awesome JSON Forms based project with the world, feel free to post it in our [JSON Forms Discourse](https://jsonforms.discourse.group/).

## JSON Forms Editor

:::info Attention

This editor is a proof of concept, is no longer maintained and is based on an older version of JSON Forms.

:::

The JSON Forms editor is a customizable visual editor for JSON Forms UI schemas.

> For example it can also be extended to serve as a visual editor for JSON Schemas.

Reviewer: ```suggestion
It can also be extended to serve as a visual editor for JSON Schemas.
```

You would like to see your project listed here?
Simply open a PR against the [JSON Forms website repository](https://github.com/eclipsesource/jsonforms2-website/blob/master/content/community/community.mdx)!

## JSON Forms Editor

:::info Attention

> This editor is a proof of concept, is no longer maintained and was based on an older version of JSON Forms.

Reviewer: ```suggestion
This editor is a proof of concept, is no longer maintained and is based on an older version of JSON Forms.
```

:::

> The JSON Forms editor is a visual editor for the JSON Forms schema and JSON Forms uischema.

Reviewer: ```suggestion
The JSON Forms editor is a customizable visual editor for JSON Forms UI schemas.
For example it can also be extended to serve as a visual editor for JSON Schemas.
```
